### PR TITLE
Added demo-facebookQuickReplies to Node demos

### PR DESCRIPTION
**Pull request by benjaminpaul**
Added a bot that demonstrates using Facebook Quick Replies and BotBuilder.

### CODE REVIEW

**Comment by msftgits** [comment]
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

**Comment by willportnoy** [comment]
@benjaminpaul thank you for the contribution.  For this repository, we have a set of standards we follow (e.g. that there are Node and C# versions of all samples, etc).  Your sample looks valuable, but may not meet the standards for this repository (e.g. a C# version).  You might consider publishing it separately to a new repo.

File: Node/demo-facebookQuickReplies/app.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    appId: process.env.MICROSOFT_APP_ID,
+    appPassword: process.env.MICROSOFT_APP_PASSWORD
+});
+server.post('/api/messages', connector.listen());
+
+var bot = new builder.UniversalBot(connector, [
+    function (session, args, next) {
+        session.beginDialog('/startdemo');
+    },
+    function (session, args, next) {
```
Comment (pcostantini) [inline Node/demo-facebookQuickReplies/app.js]: The second empty function can be removed. Also, the array enclosure. Just the first function can be passed to the UniversalBot:
````
var bot = new builder.UniversalBot(connector,
  function (session, args, next) {
    session.beginDialog('/startdemo');
  });
````

File: Node/demo-facebookQuickReplies/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "bots",
+    "chatbots",
+    "botbuilder-samples",
+    "quickreplies",
+    "facebook"
+  ],
+  "author": "Microsoft Corp",
+  "license": "MIT",
+  "dependencies": {
+    "botbuilder": "^3.7.0",
```
Comment (pcostantini) [inline Node/demo-facebookQuickReplies/package.json]: Can you update the sample to latest botbuilder version? Thanks!

**Pull request closed without merging**